requirements/development.txt

```diff
@@ -3,4 +3,5 @@ flake8==4.0.1
 notebook==6.4.12
 pylint==2.17.0
 pytest>=6.2.5
+pytest-mock>=3.10.0
 pre-commit>=2.15.0
```

Comment:
@AhmetCanSolak @keithchev I was wondering where do we add packages like `pyserial` since the vortran laser depends on it.

Reply:
`setup.cfg`, it is a core dependency IMHO